plugins/capture.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: capture
+spec:
+  version: v0.1.0
+  platforms:
+  - selector:
+      matchExpressions:
+      - {key: os, operator: In, values: [darwin, linux]}
+    uri: https://github.com/sysdiglabs/kubectl-capture/releases/download/v0.1.0/kubectl-capture.v0.1.0.tar.gz
+    sha256: "86c947a9fa9acb73eeb179e29ede20755a426a4fb96aa82375246a356f3a804f"
+    bin: "kubectl-capture"
+  shortDescription: >-
+    Triggers a Sysdig capture to troubleshoot the running pod
+  homepage: https://github.com/sysdiglabs/kubectl-capture
+  caveats: |
+    To analyze the capture file you will need to use Sysdig Inspect. You can use it as docker image directly:
+
+    ```bash
+    docker run -d -v /local/path/to/captures:/captures -p8080:3000 sysdig/sysdig-inspect:latest
+    ```
+
+    Usage:
+
+    ```bash
+    $ kubectl capture pod-name -M 45
+    ```
+
+    You can set the duration in seconds of the capture with the `-M` flag, and
+    also specify the namespace using the `-ns` or `--namespace` or use eBPF to
+    trace syscalls instead the kernel module instrumentation with `--ebpf` flag.
+  description: |
```

Comment:
Can you also include some information about the pre-requisites to use this plugin? For example, you need to be allowed/able to create privileged containers. Also, it seems to only work if the container runtime is docker? Also, what tools do you need to have installed and what kernel features need to be enabled to use the BPF capabilities?

Reply:
I added the privileged pod requirement and explained the kernel stuff needed to get sysdig up and running.

Reply:
Thanks, this is really helpful for first-time users!